dev-client/src/components/content/typography/TranslatedParagraph.tsx

```diff
@@ -31,8 +33,13 @@ export function TranslatedParagraph({
   values,
 }: TranslatedParagraphProps) {
   return (
-    <Text variant={variant}>
+    <Text variant={variant} style={styles.text}>
       <TranslatedContent i18nKey={i18nKey} values={values} />
     </Text>
   );
 }
+const styles = StyleSheet.create({
+  text: {
+    flex: 1,
```

Comment:
Ah, this again 🙃 
I don't understand this enough to know if/how it would affect anywhere else that TranslatedParagraph is used -- do you or @tm-ruxandra?

Reply:
This is not an appropriate solution. We should not be putting such a specific fix on such a general component. @paulschreiber please rework to wrap affected paragraphs in a `View` with the necessary styling.